draft-patton-cfrg-vdaf.md

```diff
@@ -176,6 +176,37 @@ is usually handled by raising an exception.
 
 # Distributed Aggregation Functions {#daf}
 
+<!--
+
+RLB: We need to elaborate the operational model a bit more here.  Maybe in a
+protocol overview section.  It seems like the major roles / steps are:
+
+* Roles: 
+  * Submitter - Knows raw metrics
+  * Aggregator - Trusted not to collued
+  * Collector - Receives aggregate at the end of the protocol (may ==
+    Aggregator)
+* Measurement process (covering both DAF and VDAF):
+  0. Raw measurement - Submitter has a measurement to submit
+  1. Sharding - Submitter computes input shares from raw measurement value
+  2. Validation - Aggregators determine that the input shares are valid
+  3. Aggregation - Aggregators compute agregate shares from input shares
+  4. Unsharding - Aggregators / Collector commpute aggregate output from
+     aggregate shares
+
+Matching the below to this framework, I see a few issues:
+
+a. The diagram indicates that the output shares are sent to aggregators
+b. The meaning of the `param` input is unclear.  Is that the working aggregate?
+c. You're missing the unsharding stage entirely
+
+The latter point is important, because the sharding and aggregation algorithms
+need to be compatible.  You kind of cover it in {{aggregatability}}, but that
+should be core part of the description -- a DAF is useless unless it can be
+unsharded.
```

Comment:
Right, the way aggregability is defined currently is that the set of output shares is an algebraic group, and the output is recovered by adding up the output shares. Would defining an explicit "unsharding" algorithm be more clear, in your opinion?